apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue

```diff
@@ -1,12 +1,13 @@
 <template>
-    <p-data-loader class="collector-options-form"
+    <p-data-loader class="collector-options-form-container"
```

Comment:
suggestion: Oh this is wrong convention.

https://cloudforet.io/ko/docs/developers/frontend/coding_convention/
<img width="881" alt="image" src="https://github.com/cloudforet-io/console/assets/26986739/ee8cffa6-4b41-4400-bc04-6667893f7ea2">

Reply:
Oops, that's only mirinae's convention. 😅

But in CollectorOptionsSection component, it uses the class name 'collector-options-form' to add css in style block.
So if you want to change the class name, please change there too!